homeassistant/components/rflink/sensor.py

```diff
@@ -354,6 +354,7 @@ def _handle_event(self, event):
 
     async def async_added_to_hass(self) -> None:
         """Register update callback."""
+        await super().async_added_to_hass()
```

Comment:
This looks like its going to cause duplicate calls based on what is in super()

I don't think we can do this here.

side note: `self.hass.data[DATA_ENTITY_LOOKUP]` all of this function makes it difficult to follow. It should use a local var